tests/pytests/test_tags.py

```diff
@@ -379,487 +380,504 @@ def test_empty_suffix_withsuffixtrie(env):
 def testEmptyValueTags():
     """Tests that empty values are indexed properly"""
 
-    env = Env(moduleArgs="DEFAULT_DIALECT 2")
+    for dialect in [2, 3, 4, 5]:
```

Comment:
We should choose how we want to address the need to execute the same tests with different dialects, between:
* The above approach (looping over the dialects in the pytest itself).
* Running the whole test-suite with different `DEFAULT_DIALECT` values, as we did until now.

Each option has its pro's and con's, but we should avoid doing both.. This can be a step towards the deprecation of the `DEFAULT_DIALECT 2` configuration that runs in case that `QUICK!=1`

Reply:
We need to define a standard way to test different dialects, but that will be part of another PR.